docs/components/camera/esp32-camera.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "Configure an esp32-camera (`viam-micro-server`)"
+linkTitle: "esp32-camera"
+weight: 33
+type: "docs"
+description: "Configure a camera connected to an esp32 board, initialized and configured using esp-idf."
+images: ["/icons/components/camera.svg"]
+tags: ["camera", "components", "micro-RDK"]
+component_description: "An `OV2640` or `OV3660` camera connected to an esp32 board."
+micrordk_component: true
+toc_hide: true
+# SMEs: Matt Perez, micro-RDK team
+---
+
+`esp32-camera` is the camera model for cameras connected to an `esp32` board, which are initialized and configured using the [`viam-micro-server` Development Setup](/installation/micro-rdk-dev/) (ESP-IDF).
+If the camera drivers are one of the following two, the camera will work with the `esp32-camera` model:
+
+- `OV2640`
+
+  - [Datasheet](https://www.uctronics.com/download/OV2640_DS.pdf)
+  - 1600 x 1200, Color, ¼” lens
+  - You can use a cam ribbon adapter to connect to your `esp32` board
+
+- `OV3660`: an [m5 camera timer module](https://docs.m5stack.com/en/unit/timercam_f)
+  - [Datasheet](https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/docs/datasheet/unit/OV3660_CSP3_DS_1.3_sida.pdf)
+  - 2048 x 1536, Color, ⅕” lens
+
```

Comment:
they are not configured using the dev setup... they are configured on this page. the micro-server provides the drivers for this. There is no operating system on the esp32 
```suggestion
`esp32-camera` is the camera model the following two cameras:

- `OV2640`

  - [Datasheet](https://www.uctronics.com/download/OV2640_DS.pdf)
  - 1600 x 1200, Color, ¼” lens
  - You can use a cam ribbon adapter to connect to your `esp32` board

- `OV3660`: an [m5 camera timer module](https://docs.m5stack.com/en/unit/timercam_f)
  - [Datasheet](https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/docs/datasheet/unit/OV3660_CSP3_DS_1.3_sida.pdf)
  - 2048 x 1536, Color, ⅕” lens

```

Reply:
@mattjperez this is meant to be used with more than just these two cameras right? What do you want to say/do you want to keep it open?

Reply:
correct. This driver is basically a wrapper around espressif's [esp32-camera drivers](https://github.com/espressif/esp32-camera), with a subset of the functionality. eventually we'd like to expose almost all of the functionality in a viam way.

Reply:
```suggestion
`esp32-camera` is the camera model that supports all cameras that work with espressif's [esp32-camera drivers](https://github.com/espressif/esp32-camera) including:

- `OV2640`

  - [Datasheet](https://www.uctronics.com/download/OV2640_DS.pdf)
  - 1600 x 1200, Color, ¼” lens
  - You can use a cam ribbon adapter to connect to your `esp32` board

- `OV3660`: an [m5 camera timer module](https://docs.m5stack.com/en/unit/timercam_f)
  - [Datasheet](https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/docs/datasheet/unit/OV3660_CSP3_DS_1.3_sida.pdf)
  - 2048 x 1536, Color, ⅕” lens

```

Reply:
changed